### DISCUSSION TITLE
Web version

### DISCUSSION THREAD

**Discussion by crumblingstatue** [yds12/tarsila#10] [Ideas]
Since tarsila is based on macroquad, it seems like a relatively low hanging fruit to have an online web version people could just run from the browser. Even if not all functionality is implemented, it could still be a nice to play around with, and get people in the mood to install the desktop version.

The web version could be hosted on GitHub pages.

**Comment by yds12** [comment]
I'm open to the idea, sounds good, and could be a way for people using OSes that are not supported (e.g. it's now a bit hard to install on old Linux distros) to use the tool.

**Comment by crumblingstatue** [comment]
I tried to get a basic wasm version working, but there seem to be problems with using certain standard library APIs that generate `__wbindgen_placeholder__` stubs which prevent the wasm from being loaded properly.
So this is perhaps not as low-hanging of a fruit as I initially hoped.
